packages/fxa-auth-server/lib/senders/emails/layouts/fxa/index.mjml

```diff
@@ -16,6 +16,7 @@
     <mj-include path="<%- locals.cssPath %>/global.css" type="css" css-inline="inline" />
     <mj-include path="<%- locals.cssPath %>/fxa/index.css" type="css" css-inline="inline" />
     <mj-include path="<%- locals.cssPath %>/locale-dir.css" type="css" />
+    <%- include('/partials/brandMessaging/index.mjml') %>
```

Comment:
Do we want the banner in SubPlat emails?

Also unless this is by design, you may want to move this out from hitting that `max-width` happening on `mj-body`. I think you can do this by removing the `body` class and either adding it to the other things that need it or wrapping everything else in another mjml element and adding the class there. Just a quick dev tools hack:
<img width="975" alt="image" src="https://github.com/mozilla/fxa/assets/13018240/5c386617-ac94-455d-9457-5000cd0f5e86">

Reply:
It seemed like the body width was very intentionally set. If you think it's okay to break out of the body width, this is fine, but I was unsure if there were other considerations around body width that I did not know about, and therefore thought keeping it inline with that was safer. I'll let you make the call though!

Reply:
Oh and that's a good question about subplat emails. FXA-7994 only calls out accounts emails, but I'll double check with Vesta about this.

Reply:
I just tried breaking out the header as suggested. This will work, but because of how we have this setup we'd either have to:
- Update every `index.mjml` file in `senders/emails/templates`  to look like `<mj section css-class="body">`
- Or, make it such that body includes the section, and the templates themselves are unaware of the `<mj section>`.


The issue appears to be that you we can't wrap simply `<%- body %>` in `layouts/index.mjml` in another `<mj section>` since it our templates already define this. 

<img width="1466" alt="image" src="https://github.com/mozilla/fxa/assets/94418270/073b66a2-ab67-43e5-8753-c689f2d38731">



Either approach would result in quite a few file changes. I don't mind making these, but please let me know what you think before I proceed.

Reply:
> It seemed like the body width was intentionally set. If you think it's okay to break out of the body width, this is fine, but I was unsure if there were other considerations around body width that I did not know about, and therefore thought keeping it inline with that was safer. I'll let you make the call though!

It is intentionally set, but it's mostly to make our text look nice since there's not a ton of content and it displays the same across devices, if I'm remembering that correctly. It seemed okay to me to break out of it for the banner, but, I think it's more of a design question than anything. If UX is cool with the banner matching the width of the text, that's fine with me 🙂

> Either approach would result in quite a few file changes. I don't mind making these, but please let me know what you think before I proceed.

Hmm can we use [`mj-wrapper`](https://documentation.mjml.io/#mj-wrapper)?

Reply:
Brilliant! 

<img width="705" alt="image" src="https://github.com/mozilla/fxa/assets/94418270/dae3078e-a5f2-44fb-a776-d8915886ce21">